### PR TITLE
First draft of README

### PR DESCRIPTION
**Pull request by FredrikFolkesson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+
+## Using Qlik Core with a license
+
+The `docker-compose.engine-and-license-service` file contains an example setup of the Qlik Associative Engine and the Qlik Licenses service.
+
+It contains the parameter `-S LicenseServiceUrl=http://licenses:9200` to the Qlik Associative Engine with the adress of the Qlik Licenses service as well as the environment variables `LICENSES_SERIAL_NBR` and `LICENSES_CONTROL_NBR` for the Qlik Licenses service.
+
+You can start it with the command `docker-compose -f docker-compose.engine-and-license-service up -d` provided that you have populated the `LICENSES_SERIAL_NBR` and `LICENSES_CONTROL_NBR` environment variables with your license.
+
+By running the command `go test test/with_license_test.go test/utils_test.go` you could then verify that with a license more than five concurent sessions could be created.
```
Comment (wennmo) [inline README.md]: typo concurent

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+You start it by running the command `docker-compose -f docker-compose.only-engine.yml up -d`
+
+You could then if you have go installed verify that the Qlik Associtaive Engine only allows five concurrent sessions by running the command `go test test/no_license_test.go test/utils_test.go`
+
+## Using Qlik Core with a license
+
+The `docker-compose.engine-and-license-service` file contains an example setup of the Qlik Associative Engine and the Qlik Licenses service.
+
+It contains the parameter `-S LicenseServiceUrl=http://licenses:9200` to the Qlik Associative Engine with the adress of the Qlik Licenses service as well as the environment variables `LICENSES_SERIAL_NBR` and `LICENSES_CONTROL_NBR` for the Qlik Licenses service.
```
Comment (wennmo) [inline README.md]: Maybe it would be good to inline the .yml file here?

Comment (FredrikFolkesson) [reply README.md]: Then it will be a manual copy of the file contents, I cant find a way to reference a yml file into a codeblock in markdown

Comment (wennmo) [reply README.md]: Ok, so perhaps insert a link to the yml instead?

**Comment by FredrikFolkesson** [comment]
@wennmo OnConnected messages would be in the qlikcore help website.

The What and How of how you configure the licensing would be in the qlikcore helpt site, and then a link to here as an example. Like this: "To see an example of a basic license configuration, see the [core-using-licenses](https://github.com/qlik-oss/core-using-licenses) repository."

**Pull request merged**